stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/MoshiChatApi.kt

```diff
@@ -0,0 +1,7 @@
+package io.getstream.chat.android.client.api2
```

Comment:
These `2`s in the package names aren't the prettiest thing ever, but we need some sort of separation of the old and the new implementation's code, and we can't just rename the old package to something `gson` related because some of its API is public. This package can be renamed later on.